Dockerfile.assisted-service

```diff
@@ -16,7 +16,7 @@ RUN cd /assisted-service/build && python3 ../tools/client_package_initializer.py
 # Build binaries
 FROM quay.io/centos/centos:stream9 as builder
 
-RUN dnf install -y gcc && dnf clean all
+RUN dnf install --enablerepo=crb -y gcc nmstate-devel && dnf clean all
```

Comment:
why not use the nmstate-static so you don't need to have nmstate.so at runtime?

Reply:
I tried to make it work with `nmstate-static`, with no success.
It kept on failing with:
`/assisted-service: error while loading shared libraries: libnmstate.so.2.1.0: cannot open shared object file: No such file or directory`

Since the dynamic Go binding work, I proceeded with that.